src/test/app/claims/converters/claimantResponseConverter.ts

```diff
@@ -0,0 +1,228 @@
+  intentionOfPaymentInFullBySetDate
+} from '../../../data/draft/paymentIntentionDraft'
+import { PaymentOption } from 'claims/models/paymentOption'
+import { MomentFactory } from 'shared/momentFactory'
+import { PaymentIntention } from 'claims/models/response/core/paymentIntention'
+import { PaymentSchedule } from 'claims/models/response/core/paymentSchedule'
+import { LocalDate } from 'forms/models/localDate'
+
+function createDraftClaimantResponseForFullRejection (): DraftClaimantResponse {
+  const draftResponse: DraftClaimantResponse = new DraftClaimantResponse()
+  draftResponse.settleAdmitted = new SettleAdmitted(YesNoOption.NO)
+  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.NO,10,100)
+  return draftResponse
+}
+function createDCRBaseForAcceptance (): DraftClaimantResponse {
+  const draftResponse: DraftClaimantResponse = new DraftClaimantResponse()
+  draftResponse.settleAdmitted = new SettleAdmitted(YesNoOption.YES)
+  draftResponse.paidAmount = new PaidAmount(PaidAmountOption.YES,10,100)
+  return draftResponse
+}
+const courtPaymentIntention = {
+  paymentOption: PaymentOption.INSTALMENTS,
+  paymentDate: MomentFactory.currentDate().add(5, 'days'),
+  repaymentPlan:  {
+    instalmentAmount: 10,
+    firstPaymentDate: MomentFactory.currentDate().add(30, 'days'),
+    paymentSchedule: PaymentSchedule.EVERY_MONTH,
+    completionDate: MomentFactory.currentDate().add(300, 'days'),
+    paymentLength: '3 Months'
+  }}
+
+const courtDecisionInstalments = {
+  'courtDecision': {
+    'paymentOption': 'INSTALMENTS',
+    'paymentDate': MomentFactory.currentDate().add(5, 'days'),
+    'repaymentPlan' : {
+      'instalmentAmount': 10,
+      'firstPaymentDate': MomentFactory.currentDate().add(30, 'days'),
+      'paymentSchedule': 'EVERY_MONTH',
+      'completionDate': MomentFactory.currentDate().add(300, 'days'),
+      'paymentLength': '3 Months'
+    }
+  }
+}
+
+function createDCRWithCourtDecisionType (claimantPI: any,decisionType: DecisionType,formaliseOption: FormaliseRepaymentPlanOption): DraftClaimantResponse {
+  const dcr: DraftClaimantResponse = new DraftClaimantResponse()
+  dcr.paidAmount = new PaidAmount(PaidAmountOption.YES,10,100)
+  dcr.formaliseRepaymentPlan = new FormaliseRepaymentPlan(formaliseOption)
+  dcr.courtDecisionType = decisionType
+  dcr.alternatePaymentMethod = DraftPaymentIntention.deserialise(claimantPI)
+  dcr.courtOfferedPaymentIntention = PaymentIntention.deserialize(courtPaymentIntention)
+  dcr.disposableIncome = 200
+  return dcr
+}
+describe('claimant response converter ',() => {
+  describe('Claimant Rejection', () => {
+    it('rejection with mediation missing ', () => {
+      expect(converter.covertToClaimantResponse(createDraftClaimantResponseForFullRejection())).to.deep.eq({
+        'type': 'rejection',
+        'amountPaid': 10
+      })
+
+    })
+
+    it('rejection with mediation ', () => {
+      const dcr = createDraftClaimantResponseForFullRejection()
```

Comment:
Rename `dcr` to `const draftClaimantResponse: DraftClaimantResponse`, same for other occurrences of `dcr`